ROBT 3341 - SCARA/scara_control.c

```diff
@@ -216,18 +233,22 @@ void inverseKinematics(double x, double y, double* joint1, double* joint2, char
 		*joint2+= 2*M_PI;
 	 }
 	 else if (*joint2 > M_PI){
-		*joint2-= 2*PI;
+		*joint2-= 2*M_PI;
 	 }
 
 	*joint1 = (*joint1*180)/M_PI;
-	*joint2 = (*joint2*180)/PI;
+	*joint2 = (*joint2*180)/M_PI;
 	//test command
 	//printf("/%lf,%lf/\n", fabs(*joint1), fabs(*joint2));
  
-	if (((fabs(*joint1)) <= MAX_ABS_THETA1_DEG) && (fabs((*joint2)) <= MAX_ABS_THETA2_DEG)){
-	   inRange = 0;
+	if (((fabs(*joint1)) <= JOINT1_MAX) && (fabs((*joint2)) <= JOINT2_MAX)){

```

Comment:
This if section looks like it does nothing and it is unclear how it works.  Should it exist?